doc/Development_Documentation/18_Tools_and_Features/28_Marketing_Settings.md

```diff
@@ -0,0 +1,16 @@
+# Marketing Settings
+
+The `Marketing Settings` give you the possibility to configure marketing-specific settings, which are:
+- Google Analytics
+- Google Search Console
+- Google Tag Manager
+
+
+### Google Analytics
+Google Analytics code is automaticaly injected during rendering the page.
+This behaviour can be disabled using:
+
+```
+$gaListener = $container->get('pimcore.event_listener.frontend.google_analytics_code');
```

Comment:
The service name was changed to the fully qualified class name in #1929 (see https://github.com/pimcore/pimcore/pull/1929/files#diff-6e09d984014c81decdca0f5a1811cadcR106). Although it is aliases to the previous name and the snippet still works, please reference the fully qualified name in the documentation (`Pimcore\Bundle\CoreBundle\EventListener\Frontend\GoogleAnalyticsCodeListener`).